topi/python/topi/nn/softmax.py

```diff
@@ -61,9 +61,11 @@ def _normalize(max_elem, expsum, *indices):
         return tvm.exp(x[indices] - max_elem[non_reduce_indices]) / expsum[non_reduce_indices]
 
     reduced_shape = tuple([dim for (i, dim) in enumerate(shape) if i != axis])
-    max_elem = tvm.compute(reduced_shape, _compute_max)
-    expsum = tvm.compute(reduced_shape, lambda *indices: _compute_expsum(max_elem, *indices))
-    return tvm.compute(shape, lambda *indices: _normalize(max_elem, expsum, *indices))
+    max_elem = tvm.compute(reduced_shape, _compute_max, name='softmax_max')
+    expsum = tvm.compute(reduced_shape, lambda *indices: _compute_expsum(max_elem, *indices),
+                         name='sofmtax_expsum')
+    return tvm.compute(shape, lambda *indices: _normalize(max_elem, expsum, *indices),
+                       name='softmax_norm')
```

Comment:
Probably prefix these names with `T_` for consistency, since all the other names are prefixed